### ISSUE TITLE
Dataloss error while trying to infere pretrained tensorflow translation  model.

### ISSUE THREAD

**Issue by fathimashaibal** [OpenNMT/OpenNMT-tf#469]
I got an error while trying to infer the TF model for German-English translation.
But the pytorch version is working fine.

DataLossError (see above for traceback): Unable to open table file /home/gpumachine/nlp-translation/OPENNMT/testdata/wmtende.model: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?
	 [[node save/RestoreV2 (defined at /home/gpumachine/anaconda3/envs/faster-rcnn/lib/python3.6/site-packages/opennmt/runner.py:423)  = RestoreV2[dtypes=[DT_INT64, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, ..., DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT, DT_FLOAT], _device="/job:localhost/replica:0/task:0/device:CPU:0"](_arg_save/Const_0_0, save/RestoreV2/tensor_names, save/RestoreV2/shape_and_slices)]]

I use tokenised data as source data and sentencepiece_model.

**Comment by guillaumekln** [comment]
Can you post your configuration file?

**Comment by fathimashaibal** [comment]
I used default configurations.

**Comment by guillaumekln** [comment]
It seems you used a SentencePiece model as a word vocabulary which is invalid.

**Comment by fathimashaibal** [comment]
Thanks for your answer. But my PyTorch translation is working fine when I using default models. Can I get the default model for TensorFlow version?.  I couldn't see the correct model. The model I downloaded was corrupted

**Comment by guillaumekln** [comment]
As said above, can you post the configuration file and the command line you used to run this model?

**Comment by guillaumekln** [comment]
Closing this issue. Feel free to reopen if you can provide your configuration files to reproduce the error.
